### PR TITLE
Jetty 10.0.x doc rename for #5206

### PR DESCRIPTION
**Pull request by WalkerWatch**

### CODE REVIEW

**Comment by joakime** [comment]
Wow, big change.

Heads up, I have several changes to doc in the jetty-home reorg PR #5186 that will likely cause conflict with this PR. (minor, and simple, but annoying)

**Review by gregw — APPROVED** [review]
I've not looked at all the files, but I like the names: op, prog, getting started.

File: jetty-documentation/src/main/asciidoc/gettingstarted-guide/introduction/what-is-jetty.adoc [hunk 1]
```diff
@@ -23,15 +23,15 @@ Eclipse Jetty is an open-source project providing an HTTP server, HTTP client, a
 
 The Jetty documentation is broken up in to four parts:
 
-* The link:{QUICKGUIDE}[Quick Start Guide] emphasizes beginning to use Jetty.
+* The link:{GSTARTEDGUIDE}[Getting Started Guide] emphasizes beginning to use Jetty.
 It provides information about what Jetty is and where you can download it, and where to find Jetty in repositories like Central Maven.
 It also provides a Quick Start guide on how to get Jetty up and running as well as an overview of how and what to configure in Jetty.
```
Comment (janbartel) [inline jetty-documentation/src/main/asciidoc/gettingstarted-guide/introduction/what-is-jetty.adoc:28]: I wouldn't capitalize Quick Start.

**Review by janbartel — APPROVED** [review]
Aside from a minor niggle with the capitalization of Quick Start, LGTM!

File: jetty-documentation/src/main/asciidoc/gettingstarted-guide/introduction/what-is-jetty.adoc (hunk 1 above)
Comment (gregw) [reply jetty-documentation/src/main/asciidoc/gettingstarted-guide/introduction/what-is-jetty.adoc:28]: I'd avoid the term Quick Start all together.   It is a getting started guide, so it provides a guide as to how to get started with jetty

Comment (WalkerWatch) [reply jetty-documentation/src/main/asciidoc/gettingstarted-guide/introduction/what-is-jetty.adoc:28]: The Quick Start terminology has been removed. The PR changes it to Getting Started.

**Pull request merged**